documentation/src/main/asciidoc/topics/proc_configuring_hotrod_percache.adoc

```diff
@@ -0,0 +1,47 @@
+[id='hotrod_percache']
+= Per-cache configuration
+
+Certain aspects of remote caches can be configured for each individual cache:
+
+* force return values
+* on-access creation configuration/template
+* near-caching
+* transaction mode
+
+.Procedure
+
+* Enable _force return values_ just for cache `a-cache`
+
+[source,java]
+----
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+...
+
+ConfigurationBuilder builder = new ConfigurationBuilder();
+builder
+  .remoteCache("a-cache")
+    .forceReturnValues(true);
+----
+
+You can also use wildcard globbing in the remote cache name to match multiple caches:
+
+.Procedure
+
+* Enable _force return values_ for all caches starting with the string `somecaches`.
```

Comment:
Could probably condense this with the preceding sentence. Maybe consider "Use wildcard globbing in the remote cache name to enable _force return values_ for all caches that start with the string `somecaches`:

Reply:
Done